doc/user-guide/schemas.rst

```diff
@@ -11,3 +11,36 @@ Schemas
         * Near-real-time "prompt" ConsDB replicates a subset of the USDF version
         * Data Release ConsDB is a snapshot of a subset of the USDF version with data pertaining to the exposures/visits in the DR
 * Schema browser
+
+* Versioning
+
+https://rubin-obs.slack.com/archives/C07QJMQ7L4A/p1730482605167509
+
+- schemas are using semantic versioning
+- should be consistent across all schemas, not just ConsDB
+
+    major: backward incompatible changes to the database objects (adding a table, deleting a column)
+
+        - except adding a table is not backwards incompatible
+
+    minor: backward compatible changes to the database objects (adding a column)
+    patch: updates or additions to semantics/metdata (units, UCDs, etc.)
+
+        - changing units can create incompatibilities
+
+    And we should say what should happen in the case of changes to primary/foreign keys.
+    Semantic neutrality: becoming non-primary is unique and anything becoming primary was already unique
+
+    - or there can be ones that are not neutral.
+
+    Think about the utility of these versions in terms of interaction with the ConsDB APIs, migrations, etc.
```

Comment:
I would think that a release of the ConsDB API should be connected to its compatible schema versions. This could be part of the release notes. I am not sure how this could be tracked automatically though. Maybe it is just something that needs to be done manually.